examples/navigation-blocking/src/app.tsx

```diff
@@ -190,11 +175,33 @@ function ImportantFormWithBlocker() {
         <button type="submit">Save</button>
       </Form>
 
-      {blockerUI[blocker.state]}
+      {blocker ? <ConfirmNavigation blocker={blocker} /> : null}
     </>
   );
 }
 
+function ConfirmNavigation({ blocker }: { blocker: Blocker }) {
+  if (blocker.state === "blocked") {
+    return (
+      <>
+        <p style={{ color: "red" }}>
+          Blocked the last navigation to {blocker.location.pathname}
```

Comment:
Moved this to a separate component so this wouldn't blow up on `blocker.location.pathname` when in an unblocked state.  everything got evaluated in the above key lookup approach.